apps/laboratory/src/components/Wagmi/WagmiPurchaseDonutSyncPermissionsTest.tsx

```diff
@@ -54,8 +65,9 @@ export function WagmiPurchaseDonutSyncPermissionsTest() {
         actions: purchaseDonutCallDataExecution,
         chain: sepolia,
         passkeyId,
-        permissions: grantedPermissions,
-        pci
+        accountAddress: address as `0x${string}`,
+        permissionsContext: (smartSessionResponse?.response?.context || '') as `0x${string}`,
```

Comment:
Can we do validation ? Throw an error if `smartSessionResponse.response.context` doesn't exist

Reply:
refactored the component and handled it